### DISCUSSION TITLE
Where is ButtonLeft from Silk.NET.Sdl ?

### DISCUSSION THREAD

**Discussion by DeafMan1983** [dotnet/Silk.NET#861] [Q&A]
Hello everyone, 

I would like to know where do I find SDL_BUTTON_LEFT from Silk.NET.Sdl?

Because I want build manuel with SDL2 Window and Renderer :D

**Comment by Perksey** [comment]
This isn't generated today, you'll have to use the individual values yourself like we've done here: https://github.dev/dotnet/Silk.NET/blob/0c5ced36ff9bc5394b7af824e77a1b079ea737a0/src/Input/Silk.NET.Input.Sdl/SdlMouse.cs#L111

Please note that our GLFW and SDL bindings are not individually maintained (they fall under the "Windowing" umbrella) and as such are very likely to be incomplete and miss a lot of functions/enums (because we've only generated the APIs we need for the most part). There is no interest from the Silk.NET team in completing our SDL bindings today.

Hope this helps!

**Reply by Perksey** [reply]
*possibly also worth noting is these bindings are slated to be removed for 3.0 as well
